mava/components/jax/building/adders.py

```diff
@@ -115,22 +102,17 @@ def required_components() -> List[Type[Callback]]:
         return [BaseTrainerInit]
 
 
-@dataclass
-class AdderSignatureConfig:
-    pass
-
-
 class AdderSignature(Component):
     """Abstract AdderSignature component defining which hooks should be used."""
 
     def __init__(
         self,
-        config: AdderSignatureConfig = AdderSignatureConfig(),
+        config: SimpleNamespace = SimpleNamespace(),
     ):
         """Component creates an adder signature for reverb adders.
 
         Args:
-            config: AdderSignatureConfig.
+            config: SimpleNamespace.
```

Comment:
I noticed inconsistencies in the past so glad about this change. Is it, make it SimpleNamespace if not config was defined instead of needing to add a class and say pass?

Reply:
Yes :slightly_smiling_face:

Reply:
The empty configs was @RuanJohn 's idea :fire:

Reply:
Nice @RuanJohn !